packages/ag-charts-community/src/chart/series/topology/mapSeriesProperties.ts

```diff
@@ -169,4 +170,10 @@ export class MapSeriesProperties extends SeriesProperties<AgMapSeriesOptions> {
 
     @Validate(OBJECT)
     override tooltip = new SeriesTooltip<AgMapSeriesTooltipRendererParams<any>>();
+
+    @Validate(STRING, { optional: true })
+    __POLYGON_STROKE?: string = undefined;
```

Comment:
Is the intent here with the naming that we want to inject theme-based values, but not allow users to set these?  (and for `__LINE_STRING_STROKE` presumably)

Reply:
Discussed offline, we'll roll with this for now.